pkg/odo/cli/project/create.go

```diff
@@ -53,6 +57,29 @@ func (pco *ProjectCreateOptions) Validate() (err error) {
 
 // Run runs the project create command
 func (pco *ProjectCreateOptions) Run() (err error) {
+
+	// Machine readable output
+	if pco.OutputFlag == "json" {
+
+		// Create the project
+		err = project.Create(pco.Client, pco.projectName, false)
+		if err != nil {
+			return err
+		}
+
+		// Get the proper machine readable format
+		output := project.GetMachineReadableFormat(pco.projectName, false, nil)
```

Comment:
I would suggest to move this to end of the function i.e. after the `pco.wait` and and `if` condition for this with `else` printing the standard user message.

Reply:
We can't. Output needs to have absolutely no information from `wait` in terms of the the spinner.. so that's why we put it before.